core/storage/suggestion/gae_models.py

```diff
@@ -626,16 +662,18 @@ class CommunityContributionStatsModel(base_models.BaseModel):
     # suggestions are offered in and the values correspond to the total number
     # of reviewers who have permission to review translation suggestions in
     # that language.
-    translation_reviewer_counts_by_lang_code = ndb.JsonProperty(required=True)
+    translation_reviewer_counts_by_lang_code = ndb.JsonProperty(
+        default={}, required=True)
     # A dictionary where the keys represent the language codes that translation
     # suggestions are offered in and the values correspond to the total number
     # of translation suggestions that are currently in review in that language.
-    translation_suggestion_counts_by_lang_code = ndb.JsonProperty(required=True)
+    translation_suggestion_counts_by_lang_code = ndb.JsonProperty(
+        default={}, required=True)
     # The total number of reviewers who have permission to review question
     # suggestions.
-    question_reviewer_count = ndb.IntegerProperty(required=True)
+    question_reviewer_count = ndb.IntegerProperty(default=0, required=True)
     # The total number of question suggestions that are currently in review.
-    question_suggestion_count = ndb.IntegerProperty(required=True)
+    question_suggestion_count = ndb.IntegerProperty(default=0, required=True)
```

Comment:
Here & above: I'm a bit concerned about retroactively changing defaults. Are there definitely not going to be any instances of this model in prod before this updated definition is used?

Reply:
This was brought up by @DubeySandeep in this [comment](https://github.com/oppia/oppia/pull/10708#discussion_r493193561), but I'll take the defaults out.

Reply:
Ack--thanks!